### DISCUSSION TITLE
Enable notifications?

### DISCUSSION THREAD

**Discussion by VictorDeGallegos** [zoidsh/meru#265] [Q&A] [answered]
I really enjoy Gmail Desktop but I would want to know if there's an option to enable notifications when I receive an email. I'm not sure this is already posible; it would be perfect ! 
Thanks

**Comment by timche** [comment, marked as answer]
In Gmail Desktop v2, notifications must be turned on in Gmail itself: https://support.google.com/mail/answer/1075549?co=GENIE.Platform%3DDesktop&hl=en&oco=0

**Reply by VictorDeGallegos** [reply]
Thank u so much, I had the notification option disabled from Google Configuration 👍🏻🥴
